src/olympia/amo/tests/test_monitor.py

```diff
@@ -126,3 +126,27 @@ def test_remotesettings_fail(self):
         )
         obtained, _ = monitors.remotesettings()
         assert '503 Server Error: Service Unavailable' in obtained
+
+    def test_cinder_success(self):
+        responses.add(
+            responses.GET,
+            'https://stage.cinder.nonprod.webservices.mozgcp.net/health',
```

Comment:
Any reason not to use the settings.CINDER_WHATEVER_URL here?

Reply:
we have to manipulate it to strip off the `/api/...` so I wanted to make sure the correct url is actually pinged (or we'd be testing the wrong url)

Reply:
Isn't the actual monitor using [settings.CINDER_SERVER_URL](https://github.com/mozilla/addons-server/pull/21953/files#diff-d1476e7a6ba82b3532601f6d7014f0a01fbbc7fef12f82cdd74ed0702b031067R238)?

Reply:
I see, you have the `/` on the path. That is very subtle. I almost would rather see something like

```
f'{settings.CINDER_SERVER_URL.replace('/api/v1', '')}/health'
```

in the test.

That makes it very clear we are removing the api part. Or split the variable to a base url and an api url. But I get it, that second option is probably too big for this PR.

Wdyt though? I find it very not obvious how the actual URL is built in the monitor and also don't really like copy pasting URLs all over the codebase.

I'm happy to approve and let you decide. If you're happy enough, then let's go, but if either of the ideas I've shared or you have another idea how to address that, also great.